examples/elixir/udp_hole_punching/puncher.exs

```diff
@@ -0,0 +1,83 @@
+defmodule Puncher do
+  @moduledoc """
+  An Ockam Worker that acts as a client in a UDP hole punching scenario.
+  """
+
+  use Ockam.Worker
+
+  require Logger
+
+  alias Ockam.{Message, Router}
+
+  @rendezvous_node "rendezvous"
+
+  @impl true
+  def handle_call(:ping_rendezvous_server, _, state) do
```

Comment:
I'm not sure to understand what's the purpose of this initial  `:ping` here.   What would be different if the first ping-pong is removed and the state machine starts by the "my address" message?